src/Plugin/Condition/PathAliasExists.php

```diff
@@ -73,11 +74,19 @@ public static function create(ContainerInterface $container, array $configuratio
   }
 
   /**
-   * {@inheritdoc}
+   * Check if a path alias exists.
+   *
+   * @param string $alias
+   *   The alias to see if exists.
+   * @param \Drupal\Core\Language\LanguageInterface|null $language
+   *   The language to use.
+   *
+   * @return bool
+   *   TRUE if the system path does not match the given alias (ie: the alias
+   *   exists).
    */
-  public function evaluate() {
-    $alias = $this->getContextValue('alias');
-    $language = $this->getContext('language')->hasContextValue() ? $this->getContextValue('language')->getId() : NULL;
+  protected function doEvaluate($alias, LanguageInterface $language = NULL) {
+    $language = is_null($language) ? NULL : $language->getId();
```

Comment:
Let's not overwrite the parameter variable $language here but rather use $lang_code instead.